packages/loader/container-loader/src/container.ts

```diff
@@ -494,6 +505,59 @@ export class Container
 
 	private readonly mc: MonitoringContext;
 
+	/**
+	 * The current generation of the Loader layer. This is used to ensure compatibility between the Loader and
+	 * other layers.
+	 */
+	private readonly generation = currentLoaderGeneration;
```

Comment:
This feels superfluous, can't we just inline `currentLoaderGeneration` wherever this is used?